glass/fields.py

```diff
@@ -520,6 +513,48 @@ def getcl(
     return cl
 
 
+def enumerate_spectra(
+    entries: Iterable[NDArray[Any]],
+) -> Iterator[tuple[int, int, NDArray[Any]]]:
+    """
+    Iterate over a set of two-point functions in :ref:`standard order
+    <twopoint_order>`, yielding a tuple of indices and their associated
+    entry from the input.
```

Comment:
Explicit "Parameters" and "Yields" missing in docstring (it would be much better to add them in this PR than in the future PRs).